examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import shutil
+import sys
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
+
+def main():
+    scriptPath = pathlib.Path(__file__).parent.absolute()  # Folder where this script is located
+    dataPath = os.path.join(scriptPath, 'data_testing')  # Folder where the data will be downloaded
```

Comment:
I don't think it's a good idea to download data relative to `__file__`.  __file__ is not your project directory; it could end up being lots of places: it could be /var/lib/python3.8/site-packages/shimming-toolbox/, it could be /home/gaspa/.local/lib/python3.8/site-packages/shimming-toolbox/, it could be the git repo (and that has its own problems to worry about: what happens if you accidentally commit the test data?), it could be /home/gaspa/.venvs/st/.

The better conventions are:

1. not to second guess the user: just work in the working directory. It's available as `os.getcwd()`, but if you just say 

```suggestion
    dataPath = 'data_testing'  # Folder where the data will be downloaded
```

it will achieve the same thing;

or 

2. Use the OS's "user app data" folder; on Windows this is something like `C:\Users\gaspa\AppData\shimmingtoolbox`; on Unix there's [the XDG spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) which says to use either `~/.local/share/shimmingtoolbox` or `~/.cache/shimmingtoolbox`.